### ISSUE TITLE
No way to pick existing IAM user on amplify configure

### ISSUE THREAD

**Issue by macginitie** [aws-amplify/amplify-cli#5718]
**Page**: [`/cli/start/install`](https://docs.amplify.aws/cli/start/install)

**Feedback**:

Does AWS Amplify require its own dedicated IAM user? Instructions force you to create one; what if you already have an IAM user with the necessary permissions? Is it possible to use it with Amplify?

**Comment by tomtaila-wesayhi** [comment]
Yeah but using amplify init seems to force you to create a new backend app in amplify. 
In my case, I have a existing IAM user AND and existing backend setup. 
`amplify configure` forces me to create a new user, whereas `init` forces me to create a new backend, both not good.

**Comment by panchohinojosa** [comment]
> Yeah but using amplify init seems to force you to create a new backend app in amplify.
> In my case, I have a existing IAM user AND and existing backend setup.
> `amplify configure` forces me to create a new user, whereas `init` forces me to create a new backend, both not good.

tldr; just ignore the console that opens up and continue. 

I found another Issue on the same topic... https://github.com/aws-amplify/amplify-cli/issues/4744 in which they clarify that `amplify configure` seems to have 2 main functionalities, create a user and configure. So as a workaround, you could add your existing user name when prompted for the new one. Then as you know the aws console webpage opens up for you to finish the aws creation of the IAM new user, but you can ignore it completely ( you can just close it ). Next in the cli, press enter to continue. This goes on to the next step thinking you finished the configuration on the webpage. Now you can continue adding your existing keys and secret. 

A fix for this would be that the cli asked you in a previous step/prompt if you have a user or you need to create one. It could then continue with the same process that it does now with the slight difference that after typing the user name: if you are creating a new user it opens up the aws console webpage and if your not, then it continues.
